GUI/widgets/plot_buttons.py

```diff
@@ -15,56 +16,64 @@ def __init__(self, plot):
 
     def initUI(self):
 
-        hbox = QHBoxLayout()
+        grid = QGridLayout()
 
         # Reset plot button
         self.reset_plot_button = QPushButton(self)
         self.reset_plot_button.setText("Reset Plot")
-        self.reset_plot_button.setStyleSheet("font-family: Helvetica; font-size: 14px; background-color : #2B26c1")
+        self.reset_plot_button.setStyleSheet(
```

Comment:
Can you notify the people working on the CSS ticket about these changes? Not sure if they already branched off.